docs/getting-started/release_package.md

```diff
@@ -0,0 +1,132 @@
+# Setup using Release package
+
+## Prerequisites
+
++ [Install docker](https://docs.docker.com/install/)
+
++ [Install kubeadm/kubectl](https://kubernetes.io/docs/setup/independent/install-kubeadm/)
+
++ [Creating cluster with kubeadm](<https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/>)
+
++ After initializing Kubernetes master, we need to expose insecure port 8080 for edgecontroller/kubectl to work with http connection to Kubernetes apiserver.
+  Please follow below steps to enable http port in Kubernetes apiserver.
+
+    ```shell
+    vi /etc/kubernetes/manifests/kube-apiserver.yaml
+    # Add the following flags in spec: containers: -command section
+    - --insecure-port=8080
+    - --insecure-bind-address=0.0.0.0
+    ```
+
++ (**Optional**)KubeEdge also supports https connection to Kubernetes apiserver. Follow the steps in [Kubernetes Documentation](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) to create the kubeconfig file.
+
+  Enter the path to kubeconfig file in controller.yaml
+  ```yaml
+  controller:
+    kube:
+      ...
+      kubeconfig: "path_to_kubeconfig_file" #Enter path to kubeconfig file to enable https connection to k8s apiserver
+  ```
+  
+  ## Cloud Vm
+ 
+  **Note**:execute the below commands as root user
+  ```
+  VERSION="v0.3.0"
+  OS="linux"
+  ARCH="amd64"
+  curl -L "https://github.com/kubeedge/kubeedge/releases/download/${VERSION}/kubeedge-${VERSION}-${OS}-${ARCH}.tar.gz" --output kubeedge-${VERSION}-${OS}-${ARCH}.tar.gz && tar -xf kubeedge-${VERSION}-${OS}-${ARCH}.tar.gz  -C /etc
+  
+  ```
+  
+  ### Generate Certificates
+  
+  RootCA certificate and a cert/key pair is required to have a setup for KubeEdge. Same cert/key pair can be used in both cloud and edge.
+  
+  ```
+   https://github.com/kubeedge/kubeedge/blob/master/build/tools/certgen.sh
+  ```
+  Copy the script from above location and execute the script
+  ```
+   bash -x ./certgen.sh genCertAndKey edge
+  ```
+  **NOTE:** The cert/key will be generated in the `/etc/kubeedge/ca` and `/etc/kubeedge/certs` respectively.
+  
+  + The path to the generated certificates should be updated in `etc/kubeedge/cloud/conf/controller.yaml`. Please update the correct paths for the following :
+      + cloudhub.ca
+      + cloudhub.cert
+      + cloudhub.key
+  
+  + Create device model and device CRDs.
+ 
+  **Note**:copy the respective yaml files in any folder, go the respective folder and execute kubectl commands as shown below
+    
+     ```
+      https://github.com/kubeedge/kubeedge/blob/master/build/crds/devices/devices_v1alpha1_devicemodel.yaml
```

Comment:
please see previous comment. give exact commands using wget to download the required yamls.

Reply:
ok